backend/data_tools/data/user_data.json5

```diff
@@ -50,6 +50,8 @@
         "PUT_BUDGET_LINE_ITEM",
         "PATCH_BUDGET_LINE_ITEM",
         "POST_BUDGET_LINE_ITEM",
+        // Delete permissions are only for local, not for prod. Users should not have delete BLI permissions in prod.
```

Comment:
@Santi-3rd @rajohnson90 what’s the right long term solution here ?  Granting extra permissions just to appease tests is problematic